### ISSUE TITLE
Fix UI of warning modal when leaving movie tunnel page

### ISSUE THREAD

**Issue by RemcoSimonides** [blockframes/blockframes#6494]
Currently it looks like this
![image.png](https://images.zenhubusercontent.com/5ed89f5adc98f2d44c81b9c0/429693c0-90a4-4208-bd2b-537999b62b56)

but it should have the layout like this
![image.png](https://images.zenhubusercontent.com/5ed89f5adc98f2d44c81b9c0/5eb14b0c-d995-410d-b9d3-f8b24866f7b5)

**Comment by Jackseed** [comment]
Blocked by #6507

**Comment by RemcoSimonides** [comment]
Actually a duplicate of #6507
Closing this one
